code/game/gamemodes/cm_initialize.dm

```diff
@@ -51,6 +51,7 @@ Additional game mode variables.
 
 /datum/game_mode
 	var/datum/mind/xenomorphs[] = list() //These are our basic lists to keep track of who is in the game.
+	var/datum/mind/queens[] = list() //List of queens
```

Comment:
This is unnecessary. Queen handling already works fine in the current code and it's all contained in the xeno proc, queen minds are also in the xenomorphs list.

Reply:
Necessary atm for reasons discussed earlier